proto/internal/temporal/server/api/persistence/v1/predicates.proto

```diff
@@ -77,3 +78,11 @@ message OutboundTaskGroupPredicateAttributes {
     repeated string groups = 1;
 }
 
+message OutboundTaskPredicateAttributes {
+    message Group {
+        string task_group = 1;
```

Comment:
It's kinda weird it's called group but can only be one type of outbound task.

Shall we make this repeated string (will certainly make and/or logic more complex)? or just call it state_ machine_task_type?

Reply:
It's not state machine task type, it's a grouping of tasks by type.
I intend in the future to support grouping start and cancel tasks together for example.